ui.py

```diff
@@ -558,57 +558,169 @@ def open_reports_window(self):
         Opens a new window to view and print reports
         :return None:
         """
-
-        # Create the pop-up window ------------------------------------
-        report_window: tk.Toplevel = tk.Toplevel(self.admin_frame)
-        report_window.title("Reports")
-        report_window.geometry("500x400")
+        # Create the pop-up window for class reports------------------------------------
+        class_report_window: tk.Toplevel = tk.Toplevel(self.admin_frame)
+        class_report_window.title("Class Reports")
+        class_report_window.geometry("500x400")
+
+        # file name text input
+        class_file_name_label: ttk.Label = ttk.Label(class_report_window, text="File Name",font=("Roboto", 12))
+        class_file_name_label.pack(pady=5)
+        class_file_name: tk.Entry = tk.Entry(class_report_window, width=20)
+        class_file_name.pack(pady=5)
+
+        # company name text input
+        class_company_name_label: ttk.Label = ttk.Label(class_report_window, text="Company Name",font=("Roboto", 12))
+        class_company_name_label.pack(pady=5)
+        class_company_name: tk.Entry = tk.Entry(class_report_window, width=20)
```

Comment:
Why don't we set the company name to "Cosmetology Class"? Since they're the ones that are going to use it, and i feel like having to type the company name every time we need a report is a little tedious.

Reply:
I disagree, I think it's better for the code to be more applicable to other applications, so I think this is fine.